### PR TITLE
CJ-8 - Update authentication service

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi @AlphaGit, @jwaimann or _another reviewer_,

I have migrated authentication services to Google+ OAuth using a customized version of [EpamSystemsBrest's Client library](https://github.com/EpamSystemsBrest/DotNetOpenAuth.Clients), it is not the best library, but it was easy to implement.

Could you review my changes?

### CODE REVIEW

**Comment by andresmoschini** [comment]
Sadly, there are problems with dependencies, and it does not work on the server, I am working on fix it.

**Comment by AlphaGit** [comment]
image

Ufffffffffffffff

File: source/CommonJobs/CommonJobs.sln [hunk 1]
```diff
@@ -83,10 +85,9 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CommonJobs.Application.Prof
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CommonJobs.Domain.Profiles", "CommonJobs.Domain.Profiles\CommonJobs.Domain.Profiles.csproj", "{A897B065-2207-4FAF-897A-31449B95F3FE}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DotNetOpenAuth.Clients", "DotNetOpenAuth.Clients\DotNetOpenAuth.Clients.csproj", "{545C1A16-75E3-469D-B6A9-9921CCD49BDD}"
+EndProject
```
Comment (AlphaGit) [inline source/CommonJobs/CommonJobs.sln]: Is this project required as part of the solution?

Comment (AlphaGit) [inline source/CommonJobs/CommonJobs.sln]: I just saw that it is being used as a dependency -- no problem.

File: source/CommonJobs/DotNetOpenAuth.Clients/UserInfo.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace DotNetOpenAuth.Clients {
+    [System.Obsolete]
```
Comment (AlphaGit) [inline source/CommonJobs/DotNetOpenAuth.Clients/UserInfo.cs]: Why creating a new class and marking it as obsolete_

**Comment by AlphaGit** [comment]
@andresmoschini Why did you add the source code to DotNetAuth.Clients instead of adding the nuget package reference?

**Comment by andresmoschini** [comment]
@AlphaGit, as you can see in fe97740dddaddf4b988f1b4f6c8745b2f0c8840b comment, I have included it and re-include dependencies in order to make it compatible with .NET 4. But it is not necessary anymore.

**Comment by andresmoschini** [comment]
Thanks for the revision JD, but I am closing it in favor of #374.

File: source/CommonJobs/DotNetOpenAuth.Clients/UserInfo.cs (hunk 2 above)
Comment (andresmoschini) [inline source/CommonJobs/DotNetOpenAuth.Clients/UserInfo.cs]: Because I was copying complete codebase (a bad decision)

**Pull request closed without merging**